_docs/_index.md

```diff
@@ -142,6 +142,9 @@ pull_image
 compress
 : enables compression og the build context using gzip
 
+output
+: sets output folder for build artifacts (format: `/drone/src/path`)
```

Comment:
The syntax for the `output` flag seems to be a bit different according to https://docs.docker.com/engine/reference/commandline/buildx_build/#output

Reply:
I didn't think that any other usage that `local` would be useful. But well, people might want to export an image as well.

Also, I see that relative paths are allowed as well. So maybe we can change this to 

> (format: `path` or `type=TYPE[,KEY=VALUE]`)

instead?

Reply:
For local trivy scans a tar export would be required as described in the issue. Making the helptext more generic sounds good.

Reply:
Oops, should've taken a closer look at the issue -_-